tensorboard/plugins/scalar/scalars_plugin_test.py

```diff
@@ -108,15 +110,32 @@ def generate_run_to_db(self, experiment_name, run_name):
           sess.run(summaries, feed_dict=feed_dict)
         sess.run(flush_op)
 
-  def testRoutesProvided(self):
-    """Tests that the plugin offers the correct routes."""
-    self.set_up_with_runs([self._RUN_WITH_SCALARS])
-    routes = self.plugin.get_plugin_apps()
-    self.assertIsInstance(routes['/scalars'], collections.Callable)
-    self.assertIsInstance(routes['/tags'], collections.Callable)
-
-  def generate_run(self, run_name):
-    subdir = os.path.join(self.logdir, run_name)
+  def with_runs(run_names):
+    """Run a test with a bare multiplexer and with a `data_provider`."""
```

Comment:
Maybe add some note about how the plugin will be passed as the first argument to the decorated function?

Reply:
Yes; thanks.